wire_test.go

```diff
@@ -206,6 +207,86 @@ func TestServerWritingResult(t *testing.T) {
 	})
 }
 
+func TestServerHandlingMultipleConnections(t *testing.T) {
+	address := TConstructMockPsqlServer(t)
+	databaseUrl := fmt.Sprintf("postgres://%s:%d", address.IP, address.Port)
+	conn, err := sql.Open("pgx", databaseUrl)
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		_ = conn.Close()
+	})
+
+	t.Run("test connectivity", func(t *testing.T) {
+		err := conn.Ping()
+		require.NoError(t, err)
+	})
+
+	t.Run("simple query execution should return rows", func(t *testing.T) {
```

Comment:
I would like to rename this subtest to just simple query. `execution should return rows` is not adding much information about the purpose of this sub test.

```suggestion
	t.Run("simple query", func(t *testing.T) {
```

Reply:
done